docs/fundamentals/single-project.md

```diff
@@ -1,35 +1,195 @@
+```
+
+At build time, the app icon is resized to the correct sizes for the target platform and device. The resized app icons are then added to your app package. App icons are resized to multiple resolutions because they have multiple uses, including being used to represent the app on the device, and in the app store.
+
+<!-- For more information, see [App icons](~/user-interface/app-icons.md). -->
+
+### Images
+
+Images can be added to your app project by dragging them to the _Resources\Images_ folder of your project, and setting their build action to **MauiImage** in the **Properties** window. This creates a corresponding entry per image in your project file:
+
+```xml
+<MauiImage Include="Resources\Images\logo.jpg" />
+```
+
+At build time, images are resized to the correct resolutions for the target platform and device. The resized images are then added to your app package.
+
+<!-- For more information, see [Images](~/user-interface/images.md). -->
+
+### Fonts
+
+True type format (TTF) and open type font (OTF) fonts can be added to your app project by dragging them into the _Resources\Fonts_ folder of your project, and setting their build action to **MauiFont** in the **Properties** window. This creates a corresponding entry per font in your project file:
+
+```xml
+<MauiFont Include="Resources\Fonts\OpenSans-Regular.ttf" />
+```
+
+At build time, the fonts are copied to your app package.
+
+For more information, see [Fonts](~/user-interface/fonts.md).
+
+### Splash screen
+
+A slash screen can be added to your app project by dragging an image into the _Resources\Images_ folder of your project, and setting the build action of the image to **MauiSplashScreen** in the **Properties** window. This creates a corresponding entry in your project file:
+
+```xml
+<MauiSplashScreen Include="Resources\Images\splashscreen.svg" />
+```
+
+At build time, the splash screen image is resized to the correct size for the target platform and device. The resized splash screen is then added to your app package.
+
+For more information, see [Splash screens](~/user-interface/splashscreen.md).
+
+### Raw assets
+
+Raw asset files, such as HTML, JSON, and videos, can be added to your app project by dragging them into the _Resources_ folder of your project (or a sub-folder, such as _Resources\Assets_), and setting their build action to `MauiAsset` in the **Properties** window. This creates a corresponding entry per asset in your project file:
+
+```xml
+<MauiAsset Include="Resources\Assets\index.html" />
+```
+
+Raw assets can then be consumed by controls, as required:
+
+```xaml
+<WebView Source="index.html" />
+```
+
+At build time, raw assets are copied to your app package.
+
+### CSS files
+
+Cascading Style Sheet (CSS) files can be added to your app project by dragging them into the _Resources_ folder of your project, and setting their build action to `MauiCss` in the **Properties** window. This creates a corresponding entry per CSS file in your project file:
+
```

Comment:
Maybe this section needs to mention Blazor or hybrid apps? Basically why a native app would need CSS.